Install Issues

I have made several attempts to install this starter. 

I start in my project folder, and I enter:

`gatsby new beach https://github.com/windedge/gatsby-tailwind-serif`

It clones the files... here is the result:

`info Creating new site from git:
https://github.com/windedge/gatsby-tailwind-serif.git

Cloning into 'beach'...
remote: Enumerating objects: 149, done.
remote: Counting objects: 100% (149/149), done.
remote: Compressing objects: 100% (107/107), done.
remote: Total 149 (delta 45), reused 136 (delta 32), pack-reused 0
Receiving objects: 100% (149/149), 917.91 KiB | 2.06 MiB/s, done.
success Created starter directory layout
info Installing packages...

yarn install v1.17.0
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > eslint-config-react-app@4.0.1" has unmet peer dependency "@typescript-eslint/eslint-plugin@1.x".
warning " > eslint-config-react-app@4.0.1" has unmet peer dependency "@typescript-eslint/parser@1.x".
warning " > eslint-config-react-app@4.0.1" has incorrect peer dependency "eslint-plugin-flowtype@2.x".
warning " > eslint-plugin-flowtype@3.11.1" has incorrect peer dependency "eslint@>=6.0.0".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
warning Your current version of Yarn is out of date. The latest version is "1.17.3", while you're on "1.17.0".
info To upgrade, run the following command:
$ brew upgrade yarn
info Initialising git in beach

info Create initial git commit in beach


   ╭───────────────────────────────────────╮
   │                                       │
   │   Update available 2.7.20 → 2.7.21    │
   │   Run npm i -g gatsby-cli to update   │
   │                                       │
   ╰───────────────────────────────────────╯

'
Then I

`cd beach` 

to enter the folder, and 

` npm run dev
npm ERR! Invalid version: "1.0"

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/brianeller/.npm/_logs/2019-07-29T22_23_36_185Z-debug.log

Next step is: 
`gatsby develop
success open and validate gatsby-configs - 0.026 s

 ERROR 

Error in "/Users/brianeller/Projects/beach/node_modules/gatsby-plugin-offline/gatsby-node.js": Unexpected token }



  Error: /Users/brianeller/Projects/beach/node_modules/core-js/library/modules/_
  is-object.js:3
  });
  ^
  SyntaxError: Unexpected token }
  
  - v8-compile-cache.js:226 NativeCompileCache._moduleCompile
    [beach]/[v8-compile-cache]/v8-compile-cache.js:226:18
  
  - v8-compile-cache.js:172 Module._compile
    [beach]/[v8-compile-cache]/v8-compile-cache.js:172:36
  
  - v8-compile-cache.js:159 require
    [beach]/[v8-compile-cache]/v8-compile-cache.js:159:20
  
  - v8-compile-cache.js:178 Module._compile
    [beach]/[v8-compile-cache]/v8-compile-cache.js:178:30
  
  - v8-compile-cache.js:159 require
    [beach]/[v8-compile-cache]/v8-compile-cache.js:159:20
  
⠋ load plugins`

and halts.

I have yet to complete an install using this starter. I am new to Gatsby, so I'll ask for help.

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@windedge/gatsby-tailwind-serif",
   "description": "Gatsby starter styled with Tailwind",
-  "version": "1.0",
+  "version": "1.0.0",
   "author": "@windedge",
   "dependencies": {
     "classnames": "^2.2.6",

